### PR TITLE
fix(analytics): 12145 - change analitics header

### PR DESCRIPTION
**Pull request by IlyaIzr**
image
Analytics panels now have static names and disaster info moved into panel body

### CODE REVIEW

File: src/core/localization/translations/en/common-messages.json [hunk 1]
```diff
@@ -68,12 +68,12 @@
       "init": "Select Geometry",
       "loading": "Loading analytics...",
       "error": "Error",
-      "other": "Analytics",
+      "header_title": "Analytics",
```
Comment (Komzpa) [inline src/core/localization/translations/en/common-messages.json:71]: did you also regenerate Spanish?

Comment (propakov) [reply src/core/localization/translations/en/common-messages.json:71]: it's not in the main branch, need to add it.

Comment (propakov) [reply src/core/localization/translations/en/common-messages.json:71]: @IlyaIzr could you please approve it https://github.com/konturio/disaster-ninja-fe/pull/151 ? 
and then we can fix key in spanish too

Comment (propakov) [reply src/core/localization/translations/en/common-messages.json:71]: @IlyaIzr merged

Comment (IlyaIzr) [reply src/core/localization/translations/en/common-messages.json:71]: changed it in spanish file too, resolving this one

File: src/components/Analytics/AnalyticsPanelHeader/AnalyticsPanelHeader.tsx [hunk 2]
```diff
@@ -56,16 +51,12 @@ const AnalyticsPanelHeader = ({
...
 
   return statesToComponents({
-    init: () => <Text type="heading-m">{i18n.t(messages.init)}</Text>,
-    loading: () => <Text type="heading-m">{i18n.t(messages.loading)}</Text>,
-    error: () => <Text type="heading-m">{i18n.t(messages.error)}</Text>,
+    loading: () => <Text type="heading-m">{i18n.t(loadingMessage)}</Text>,
+    error: () => null,
     ready: () =>
       ({
         event: <PanelHeading event={(focusedGeometry?.source as any).meta} />,
```
Comment (propakov) [inline src/components/Analytics/AnalyticsPanelHeader/AnalyticsPanelHeader.tsx:58]: could we use actual types in casting here?
```
        event: <PanelHeading event={(focusedGeometry?.source as GeometrySourceEvent).meta} />,
        boundaries: <Text type="heading-m">{(focusedGeometry?.source as GeometrySourceBoundaries).meta}
```

**Review by propakov — APPROVED** [review]
LGTM! 
pls check one moment with types, maybe it's suitable

**Pull request merged**